### DISCUSSION TITLE
Simple Example

### DISCUSSION THREAD

**Discussion by mshzsh** [vizia/vizia#418] [Questions] [answered]
I'm looking for a guide as I've read the documentation but haven't found answers to some of my questions. Could someone provide an example code where, for instance, there's a text input, and when you type text into it and press the Enter key, the text gets saved to the system clipboard, and the program closes?

**Comment by YouKnow-sys** [comment, marked as answer]
You should take a look at `Textbox` view and its `on_submit` for getting input from user, ~for saving the text into clipboard you should take a look at other crates that offer such thing, because its not something that vizia offer~ nevermind, vizia actually support doing such thing using the `set_clipboard` method on cx.
for a more in-depth example you can take a look at this code that I wrote using vizia to get to know it better [password-generator](https://github.com/YouKnow-sys/password-generator/blob/master/src/gui/mod.rs) it have both text input and the ability to save the text to clipboard.

**Reply by mshzsh** [reply]
Thank you!

**Reply by YouKnow-sys** [reply]
> Thank you!

If this answered your question please select it as discussion answer.
